terraform/groups/lambda/profiles/development-eu-west-2/cidev/vars

```diff
@@ -0,0 +1,12 @@
+# ------------------------------------------------------------------------------
+# Environment
+# ------------------------------------------------------------------------------
+
+environment = "cidev"
+aws_profile = "development-eu-west-2"
+cron_schedule = "cron(0 1 * * ? *)"
+aws_region = "eu-west-2"
+release_bucket_name = "development-eu-west-2.release.ch.gov.uk"
```

Comment:
Also unsure if this is still the correct release bucket. I think it is but worth double checking against some ECS builds